src/withStyles.jsx

```diff
@@ -56,9 +56,21 @@ export function withStyles(
         this.maybeCreateStyles();
       }
 
+      getDirection() {
```

Comment:
Nit: Maybe just make this `isRTL()` since that's how it's used in both places below?

```js
const direction = this.context[CHANNEL] && this.context[CHANNEL].getState();

return direction === DIRECTIONS.RTL;
```